config/identify_system.go

```diff
@@ -46,10 +48,15 @@ func identifySystem(config ServerConfig) (systemType string, systemScope string,
 		if systemID == "" {
 			systemID = config.CrunchyBridgeClusterID
 		}
-	} else if (config.AivenServiceID != "") || systemType == "aiven" {
-		systemType = "aiven"
+	} else if (config.AivenProjectID != "" && config.AivenServiceID != "") || systemType == "aiven" {
+		systemTypeFallback = "aiven"
```

Comment:
Looking at how this works for AWS, this is not quite right--still reviewing, but I'll update.